runtime/onert/backend/trix/DevContext.h

```diff
@@ -55,6 +55,64 @@ class DevContext
 
   npudev_h getDev() { return _dev_handle; }
 
+  template <typename T> void setDataInfo(tensors_data_info *info, std::vector<T *> &tensors)
```

Comment:
I think there is no reason to use `function template`

Reply:
This function supports both case of `const IPortableTensor` as input and `IPortableTensor` as output. So I use template function.

Reply:
I think that a function `void setDataInfo(tensors_data_info *info, const std::vector<const IPortableTensor *> &tensors)` supports both cases. Is there a problem?

Reply:
I don't think that const Variable will be compatible with Variable........ Am I wrong..?

```
/opt/test/runtime/onert/backend/trix/ops/BulkLayer.cc:85:40: error: cannot convert 'std::vector<onert::backend::IPortableTensor*>' to 'const std::vector<const onert::backend::IPortableTensor*>&'
   85 |   _dev_context->setDataInfo(&out_info, _outputs);
      |                                        ^~~~~~~~
      |                                        |
      |                                        std::vector<onert::backend::IPortableTensor*>
In file included from /opt/test/runtime/onert/backend/trix/ops/BulkLayer.h:21,
                 from /opt/test/runtime/onert/backend/trix/ops/BulkLayer.cc:17:
/opt/test/runtime/onert/backend/trix/ops/../DevContext.h:58:89: note:   initializing argument 2 of 'void onert::backend::trix::DevContext::setDataInfo(tensors_data_info*, const std::vector<const onert::backend::IPortableTensor*>&)'
   58 |   void setDataInfo(tensors_data_info *info, const std::vector<const IPortableTensor *> &tensors)
      |                                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~
make[3]: *** [runtime/onert/backend/trix/CMakeFiles/onert_backend_trix.dir/build.make:102: runtime/onert/backend/trix/CMakeFiles/onert_backend_trix.dir/ops/BulkLayer.cc.o] Error 1
```

Reply:
I did't know they are different types. How about using a way (template <class const_iter>
void f(const_iter b, const_iter e) {} ) in [link](https://code-examples.net/en/q/2013e4). It's not madatory, so you don't have to.

Reply:
I've viewed the link, but the `setDataInfo` function requires a total size of the tensor vector. If I pass the const_iterator of the vector, I have to assign the total tensor size outside the `setDataInfo` function. It is not a pretty code I think. Why don't we just keep it?

Reply:
OK. The suggested way doesn't look good either.